farcy/handlers.py

```diff
@@ -183,3 +183,39 @@ def _process(self, filename):
             retval[offense['location']['line']].append(
                 '{cop_name}: {message}'.format(**offense))
         return retval
+
+
+class JSXHint(ExtHandler):
```

Comment:
Can you move this such that the class name is sorted relative to its sibling classes? If we're going to have big files (common in python) it's nice to have class and variables names sorted.